common/tzif.ksy

```diff
@@ -0,0 +1,279 @@
+meta:
+  id: tzif
+  title: The Time Zone Information Format (TZif)
+  xref:
+    mime:
+      - application/tzif
+      - application/tzif-leap
+    rfc: 8536
```

Comment:
You said in `/doc` that

> It is used by most **UNIX** systems to calculate local time for a time zone

..., so you might tag the format as `unix`-related:

```suggestion
    rfc: 8536
  tags:
    - unix
```

Note: in https://github.com/kaitai-io/kaitai_struct_formats/issues/361#issuecomment-753707340 shares @dgelessus his opinion whether to keep both `linux` and `unix` tags (currently existing specs in the format gallery [only use `linux`](https://github.com/kaitai-io/kaitai_struct_formats/search?q=tags+linux)).

Reply:
How about `nix` tag?

Reply:
It's definitely not Linux-specific, and not really Unix-specific either. Other systems such as Android and iOS use TZif files. There are also TZif readers for other software systems, for example Python's `zoneinfo` module and the timezone support in PHP.

Reply:
> It's definitely not Linux-specific, and not really Unix-specific either. Other systems such as Android and iOS use TZif files. There are also TZif readers for other software systems, for example Python's `zoneinfo` module and the timezone support in PHP.

OK, then it probably doesn't make sense to add it.